components/Banner/tests/unit/Theming/MDCBannerViewThemingTests.m

```diff
@@ -0,0 +1,135 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <XCTest/XCTest.h>
+
+#import "MaterialBanner+Theming.h"
+#import "MaterialBanner.h"
+#import "MaterialContainerScheme.h"
+
+static CGFloat const kBannerViewTextViewOpacityDefault = (CGFloat)0.87;
```

Comment:
Please add comments to these constants. It looks like they were missed when you added comments to their equivalents in the Theming Extension.

Reply:
Addressed in bab2aff59941b6a8f32a7fc7ca7efb5b742ec680.